mlir/test/Dialect/LLVMIR/mem2reg.mlir

```diff
@@ -1033,7 +1033,7 @@ llvm.func @scalable_vector() -> i16 {
 llvm.func @scalable_llvm_vector() -> i16 {
   %0 = llvm.mlir.constant(1 : i32) : i32
   // CHECK: llvm.alloca
-  %1 = llvm.alloca %0 x !llvm.vec<? x 4 x ppc_fp128> : (i32) -> !llvm.ptr
+  %1 = llvm.alloca %0 x vector<[4] x !llvm.ppc_fp128> : (i32) -> !llvm.ptr
```

Comment:
Interesting... I didn't expect a scalable vector with this element type...

Reply:
The element type shouldn't matter, right? "Scalability" is a fairly abstract concept.